theano/tensor/tests/test_opt.py

```diff
@@ -5675,18 +5675,19 @@ def test_local_useless_split():
 
 def test_local_flatten_lift():
     for i in xrange(1, 4):
-        op = tensor.Flatten(i)
         x = tensor.tensor4()
-        out = op(T.exp(x))
+        out = tensor.flatten(T.exp(x), i)
         assert out.ndim == i
         mode = compile.mode.get_default_mode()
         mode = mode.including('local_flatten_lift')
```

Comment:
use this mode to fix the travis problem:
mode=mode.including('local_reshape_lift')